src/storybook_stories/stories.css

```diff
@@ -0,0 +1,244 @@
+/*
```

Comment:
Deze hele file mag je mijns inziens gewoon deleten voordat je gaat pushen, moeten we nog ff naar kijken hoe we dit netjes kunnen oplossen, volgens mij heeft dit iets te maken met 't dev script dat ook iets met storybook doet waardoor die css weg wordt geschreven, anders moeten we die storybook folder in de gitignore zetten.

Reply:
Daar weet @deannabosschert wss wel wat op te verzinnen